transformer_lens/components.py

```diff
@@ -298,7 +301,9 @@ def __init__(
         # See HookedTransformerConfig for more details.
         if self.cfg.positional_embedding_type == "shortformer":
             # This tracks the input to the keys and queries, which is resid_pre + pos_embeds
-            self.hook_attn_input = HookPoint()  # [batch, pos, d_model]
+            # The dimension of these depends on whether cfg.use_attn_input is False (1st) or True (2nd)
+            self.hook_attn_input_q = HookPoint()  # [batch, pos, d_model] OR [batch, pos, head_index, d_model]
```

Comment:
Oh man, the shortformer bits of this are really ugly. I weakly lean towards just having this feature only apply to non shortformer positional embeddings...?